aana/tests/units/test_image_chat_dialog.py

```diff
@@ -0,0 +1,169 @@
+# ruff: noqa: S101
+
+from pathlib import Path
+
+import pytest
+
+from aana.core.models.image import Image
+from aana.core.models.image_chat import (
+    ImageChatDialog,
+    ImageChatMessage,
+    ImageContent,
+    TextContent,
+)
+
+IMAGE_PATH = Path("aana/tests/files/images/Starry_Night.jpeg")
```

Comment:
Use `path = resources.path("aana.tests.files.images", "Starry_Night.jpeg")` instead

Reply:
And maybe use fixtures. You can create a fixture for Image object instead of creating it in the tests.

Reply:
changed